docs/design.html

```diff
@@ -543,25 +544,25 @@ <h4><a id="design_compactiondetails" href="#design_compactiondetails">Log Compac
     (assuming 1k messages).
     </ol>
     <p>
-    <h4><a id="design_compactionconfig" href="#design_compactionconfig">Configuring The Log Cleaner</a></h4>
+    <h4 class="anchor-heading"><a id="design_compactionconfig" class="anchor-link"></a><a href="#design_compactionconfig">Configuring The Log Cleaner</a></h4>
 
     The log cleaner is enabled by default. This will start the pool of cleaner threads.
     To enable log cleaning on a particular topic, add the log-specific property
-    <pre class="brush: text;"> log.cleanup.policy=compact</pre>
+    <pre class="language-text"> log.cleanup.policy=compact</code></pre>
```

Comment:
Seems we are missing a opening `<code>` tag? Similar the next two changes below.

Reply:
ack